src/test/java/nl/tudelft/contextproject/controller/WaitingControllerTest.java

```diff
@@ -37,9 +39,10 @@ public static void tearDownAfterClass() {
 	
 	/**
 	 * Create a new instance of the controller for each test.
+	 * @throws FileNotFoundException This should not happen.
```

Comment:
Why shouldn't this happen? If you can guarantee this will never happen you can use SneakyThrows